src/state.ts

```diff
@@ -23,8 +29,8 @@ class State<H = unknown> {
     return res;
   }
 
-  runEffects(): void {
-    let effects = this[effectsSymbol];
+  runEffects(phase: EffectsSymbols): void {
```

Comment:
This function is meant to be accessible from any library that wants to mixin Haunted behavior into a component, so we can't force a symbol here.  I'd like to keep `runEffects` to mean "run useEffects". So may be you can make this function be `_runEffects` and then create a `runEffects` and `runLayoutEffects` that pass the appropriate symbol to `_runEffects`.